apps/codeforafrica/src/components/NavList/index.js

```diff
@@ -0,0 +1,47 @@
+import { styled } from "@mui/material/styles";
+import PropTypes from "prop-types";
+import * as React from "react";
+
+const NavListRoot = styled("ul", {
+  shouldForwardProp: (prop) => prop !== "direction",
+  slot: "Root",
+  overridesResolver: (props, styles) => {
+    const { direction } = props.ownerState;
+    return [styles.root, styles[direction]];
+  },
+})(({ ownerState }) => ({
+  display: "flex",
+  ...(ownerState.direction === "column" && {
+    flexDirection: "column",
+    justifyContent: "flex-start",
+    alignItem: "flex-start",
+  }),
+  ...(ownerState.direction === "row" && {
+    flexDirection: "row",
+    justifyContent: "flex-end",
+    alignItem: "center",
+  }),
+}));
+
+const NavList = React.forwardRef(function NavList(props, ref) {
+  const { direction = "column", children, ...others } = props;
+  const ownerState = { ...others, direction };
```

Comment:
```suggestion
  const ownerState = { direction };
```
We only seem to use `direction`. Lets pass just that.

Reply:
This hasn't been addressed.